docs/build/differences-vs-evm/index.md

```diff
@@ -53,9 +65,9 @@ If you are a Solidity developer, it's recommended to read Cadence's [Guide for S
   - [Tutorial](https://cadence-lang.org/docs/tutorial/marketplace-setup)
   - [NFT Storefront](https://github.com/onflow/nft-storefront/) is an example marketplace standard
 
-### Transactions and Scripts
+## Transactions and Scripts
 
-You can interact with the state on most other blockchains by cryptographically authorizing smart contract function calls. On Flow, transactions are more complex and are pieces of Cadence code. This means that any number of contracts and function calls can be composed together atomically to mutate the blockchain state.
+You can interact with the state on most other blockchains by cryptographically authorizing smart contract function calls. On Flow, transactions are more complex and are written with Cadence code. This means that any number of contracts and function calls can be composed together atomically to mutate the blockchain state.
```

Comment:
can we say this a bit differently? Maybe something like:
"On Flow, transactions offer rich functionality through Cadence code. This allows you to seamlessly combine multiple contracts and function calls into a single transaction that updates the blockchain state - all executing together as one unified operation."